core/processor/ProcessorParseContainerLogNative.h

```diff
@@ -44,11 +59,21 @@ class ProcessorParseContainerLogNative : public Processor {
     std::string containerSourceKey = "_source_"; // 容器来源字段
     std::string containerLogKey = "content"; // 容器日志字段
 
-    bool ProcessEvent(const StringView& containerType, PipelineEventPtr& e);
-    void AddDockerJsonLog(char ** data ,const StringView& key, const StringView& value, LogEvent& targetEvent);
-    void AddLog(const StringView& key, const StringView& value, LogEvent& targetEvent, bool overwritten = true);
-    bool ContainerdLogLineParser(LogEvent& sourceEvent, PipelineEventPtr& e);
-    bool DockerJsonLogLineParser(LogEvent& sourceEvent, PipelineEventPtr& e);
+    bool ProcessEvent(PipelineEventGroup& logGroup,
+                      const StringView containerType,
+                      PipelineEventPtr& e,
+                      ContainerStdoutKeyStringBuffer* containerStdoutKeyStringBuffer);
+    void AddDockerJsonLog(char** data, const StringBuffer key, const StringView value, LogEvent& targetEvent);
```

Comment:
1. StringView不需要const修饰
2. char**不太常用，还是改成两个参数吧，加一个index记录当前位置

根据之前的描述，可以直接改成
```suggestion
    void AddDockerJsonLog(char* data, size_t &index, StringView key, StringView value, LogEvent& targetEvent);
```